.travis.yml

```diff
@@ -70,20 +70,14 @@ jobs:
       rust: stable
     - <<: *rust
       rust: beta
-      branches:
-        only:
-          - staging
-          - trying
+      if: branch = staging || branch = staging
```

Comment:
Shouldn't this be `branch = staging || branch = trying`?

Reply:
true -- not sure if the OR syntax works at all though :)